### ISSUE TITLE
Please give us TemplaVoilaPlus 8

### ISSUE THREAD

**Issue by typo3max** [pluspol-interactive/templavoilaplus#334]

**Comment by typo3max** [comment]
i Beg

**Comment by jpmschuler** [comment]
There is a 8 beta which is quite fine?

**Comment by typo3max** [comment]
But not open release!

**Comment by Outdoorsman** [comment]
I tried `8.0.0-beta.2` [from packagist](https://packagist.org/packages/templavoilaplus/templavoilaplus#8.0.0-beta.2) but everything disappears on the frontend and it seems the templates weren't being found. Is there an upgrade step I need to do when moving to version 8?

Secondly, I'm also hesitant to upgrade to a beta version with production sites for my clients and they are still stuck at TYPO3 version 9 because of this :( I saw the milestone list of issues for version 8 still and it didn't make me feel like it was a good idea for production???

**Comment by jpmschuler** [comment]
Yes there is an upgrade available in the backend. A lot of stuff changed (e.g. mapping of templates moved from DB to files).
If everthing is still empty after that probably you missed to change the userFunc for page rendering in TypoScript:
`page.10.userFunc = Tvp\TemplaVoilaPlus\Controller\Frontend\FrontendController->renderPage`
